source/agora/consensus/protocol/Nominator.d

```diff
@@ -186,6 +193,25 @@ extern(D):
         this.restoreSCPState();
         this.nomination_interval = nomination_interval;
         this.acceptBlock = externalize;
+        this.envelope_timer = this.taskman.setTimer(10.msecs,
```

Comment:
Is it worth making this configurable? I think it was the comment I had last time.

Reply:
Actually I gave it some thought and maybe it should be merged with the nomination timer ?

Reply:
But the nomination timer is how frequently we check if it is time to nominate where as this is how often to check for envelopes that need processing. The envelope checking probably should be more frequent. Or is the overhead of a second timer the issue?

Reply:
Not worried about the overhead, but worried about race conditions

Reply:
We have `isNominating` set which we could check before processing envelopes.

Reply:
That sounds a bit brittle, but let's leave that discussion for another day.